src/main/resources/schema.graphqls

```diff
@@ -1,6 +1,6 @@
 scalar JSON
 
-type Run {
+type Run @key(fields: "runName") {
```

Comment:
should we swap `runName` in as the the `ID!` field?

Reply:
good call